nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GetHTTP.java

```diff
@@ -102,6 +96,11 @@
     + "once the content has been fetched from the given URL, it will not be fetched again until the content on the remote server changes. Note that due to limitations on state "
     + "management, stored \"last modified\" and etag fields never expire. If the URL in GetHttp uses Expression Language that is unbounded, there "
     + "is the potential for Out of Memory Errors to occur.")
+@DynamicProperties({
+    @DynamicProperty(name = "header_XYZ", value = "The Expression Language to be used to populate the header value", description = "The additional headers to be sent by the processor " +
+            "whenever making a new HTTP request. \n " +
+            "Setting this the name to XYZ and value to ${attribute} will result on the header 'XYZ: attribute_value' being sent to the HTTP endpoint"),
```

Comment:
Minor things! I am English as an Only Language, so you are totally winning in this area!